docs/solutions/workflow/gt-modify-no-c-flag-silent-unstaged-miss.md

```
@@ -0,0 +1,86 @@
+the unstaged file.
+
+The output "X files changed" refers to the amended commit's cumulative diff
+relative to its parent — not to what was just added. This makes it look like
+success when the edit was missed entirely.
+
+## Solution
+
+Two correct patterns:
+
+**Option A — stage first, then amend:**
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)